### PR TITLE
T2: Displaying diff that allowed the t2-ixia to pass.

### PR DESCRIPTION
**Pull request by rraghav-cisco**
@abdosi , @SuvarnaMeenakshi for Viz

@selldinesh , @kamalsahu0001, @isgmano : for Viz.
### Description of PR
Summary:
Listing out the diff that allowed the t2-ixia script to run. This is a feedback kind of diff, for Keysight team, to implement in the scripts, both current and future.

### Type of change

- [X] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### CODE REVIEW

**Comment by abdosi** [comment]
@selldinesh , @kamalsahu0001, @isgmano  please help with review of this on priority

**Comment by rraghav-cisco** [comment]
@vmittal-msft : for viz.

File: tests/common/snappi_tests/snappi_fixtures.py [hunk 1]
```diff
@@ -549,7 +543,7 @@ def snappi_dut_base_config(duthost_list,
                         for i, sp in enumerate(snappi_ports) if sp['location'] in tgen_ports]
     pytest_assert(len(set([sp['speed'] for sp in new_snappi_ports])) == 1, 'Ports have different link speeds')
     [config.ports.port(name='Port {}'.format(sp['port_id']), location=sp['location']) for sp in new_snappi_ports]
-    speed_gbps = int(new_snappi_ports[0]['speed'])/1000
```
Comment (rraghav-cisco) [inline tests/common/snappi_tests/snappi_fixtures.py:552]: Another PR addressing this.

File: tests/snappi_tests/multidut/pfc/files/multidut_helper.py [hunk 2]
```diff
@@ -25,9 +25,9 @@
 dut_port_config = []
 PAUSE_FLOW_NAME = 'Pause Storm'
 TEST_FLOW_NAME = 'Test Flow'
-TEST_FLOW_AGGR_RATE_PERCENT = 45
```
Comment (rraghav-cisco) [inline tests/snappi_tests/multidut/pfc/files/multidut_helper.py:28]: Can we see the backplane speed in config db ?

File: tests/snappi_tests/multidut/pfc/files/multidut_helper.py [hunk 3]
```diff
@@ -264,6 +265,7 @@ def run_pfc_test(api,
     # Verify PFC pause frame count on the DUT
     verify_pause_frame_count_dut(duthost=duthost,
                                  test_traffic_pause=test_traffic_pause,
+                                 global_pause=True,
```
Comment (rraghav-cisco) [inline tests/snappi_tests/multidut/pfc/files/multidut_helper.py:268]: Addressed in 12202

File: tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py [hunk 4]
```diff
@@ -26,7 +26,9 @@ def test_pfc_pause_single_lossless_prio(snappi_api,                     # noqa:
                                         duthosts,
                                         line_card_choice,
                                         linecard_configuration_set,
-                                        get_multidut_snappi_ports):       # noqa: F811
+                                        get_multidut_snappi_ports,      # noqa: F811
+                                        lossless_prio_list,             # noqa: F811
```
Comment (rraghav-cisco) [inline tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py:30]: Addressed in 12004

File: tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py [hunk 5]
```diff
@@ -64,9 +66,7 @@ def test_pfc_pause_single_lossless_prio(snappi_api,                     # noqa:
         pytest_require(False, "Need Minimum of 2 ports for the test")
 
     snappi_ports = get_multidut_tgen_peer_port_set(line_card_choice, snappi_port_list, config_set, 2)
-    tgen_ports = [port['location'] for port in snappi_ports]
     testbed_config, port_config_list, snappi_ports = snappi_dut_base_config(dut_list,
-                                                                            tgen_ports,
```
Comment (rraghav-cisco) [inline tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py:69]: Addressed in 12004

File: tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py [hunk 6]
```diff
@@ -175,7 +175,8 @@ def test_pfc_pause_multi_lossless_prio(snappi_api,                  # noqa: F811
 
 
 @pytest.mark.disable_loganalyzer
-@pytest.mark.parametrize('reboot_type', ['warm', 'cold', 'fast'])
+@pytest.mark.parametrize('reboot_type', ['cold'])
```
Comment (rraghav-cisco) [inline tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py:178]: there is a PR for skip _warm reboot.

**Comment by abdosi** [comment]
@rraghav-cisco @sdszhang Can we close this PR ? It seems not needed.

**Comment by rraghav-cisco** [comment]
Closing, since it is only for reference.

**Pull request closed without merging**